FastText/src/models/pretrain_lm.jl

```diff
@@ -0,0 +1,52 @@
+using DelimitedFiles
+
+"""
+ULMFiT - LANGUAGE MODEL
+
+    LanguageModel(load_pretrained::Bool=false, vocabpath::String="vocabs/lm_vocab.csv";kw...)
+
+The Language model structure for ULMFit is defined by 'LanguageModel' struct.
+It contains has two fields:
+    vocab   : vocabulary, which will be used for language modelling
+    layers  : embedding, RNN and dropout layers of the whole model
+In this language model, the embedding matrix used in the embedding layer
+is same for the softmax layer, following Weight-tying technique.
+The field 'layers' also includes the Variational Dropout layers.
+It takes several dropout probabilities for different dropout for different layers.
+
+[Usage and arguments are discussed in the docs]
+There are several keyword argunments to set the dropout probabilities
+of the layers of model checkout those arguments in the docs.
+# Example:
+
+julia> lm = LanguageModel()
+"""
+mutable struct LanguageModel
```

Comment:
```suggestion
struct LanguageModel
```
Both fields of this struct are abstract types, so we'll have to think about that sooner or later. Until then, can this be immutable?